File: 算法基础课(修订版)/循环/P14077/data/user_acm.py
class Solution:
    def solve(self, matrix, x1, y1, x2, y2):
        # 请在这里实现
        return 0


if __name__ == "__main__":
    n, m = map(int, input().split())
    matrix = []
    for _ in range(n):
        matrix.append(list(map(int, input().split())))

    q = int(input())
    solution = Solution()
    for _ in range(q):
        x1, y1, x2, y2 = map(int, input().split())
        print(solution.solve(matrix, x1, y1, x2, y2))

File: 算法基础课(修订版)/笔试拿分技巧/P14163/user_acm.cc
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
    string solve(vector<int>& a) {
        // 请在这里实现
        return "";
    }
};

int main() {
    int n;
    cin >> n;
    vector<int> a(n);
    for (int i = 0; i < n; ++i) {
        cin >> a[i];
    }

    Solution solution;
    cout << solution.solve(a);
    return 0;
}
